src/features/serveStatic/SetupIPCRendererHandler.tsx

```diff
@@ -0,0 +1,10 @@
+import React, { memo } from 'react'
+import { ipcRenderer } from 'electron'
+
+const SetupIPCRendererHandler = memo(function SetupIPCRendererHandler() {
+  ipcRenderer.send('app-ready')
+
+  return <></>
+})
```

Comment:
Please use hook instead of component without output

```js
export const useSetupIPCHandler = () => {
  useEffect(() => {
      ipcRenderer.send('app-ready')
  }, [])
}
```

Reply:
It has been updated.